package.json

```diff
@@ -15,10 +15,11 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/apache/cordova-plugin-inappbrowser"
+    "url": "github:apache/cordova-plugin-inappbrowser"
```

Comment:
Shouldn't this be the complete shorthand syntax?
`"repository": "github:apache/cordova-plugin-inappbrowser"` - personally I'd drop the `github:` part, too. Recent npm versions should be intelligent enough to resolve this to GitHub

Reply:
I just wrote a script that updated the existing URL. I didn't think of shortening the entire field to a string. 

Yeah we can shorten it more.

https://docs.npmjs.com/files/package.json#repository

I wonder what is the order of priority if you dropped GitHub from the string.

```
"repository": "npm/npm"
"repository": "github:user/repo"
"repository": "gist:11081aaa281"
"repository": "bitbucket:user/repo"
"repository": "gitlab:user/repo"
```

Would be bad if someone claimed the other services and NPM decided to use one of the other services.

Reply:
That *could* happen, but I don't believe it's that probable ;)  
Having github: doesn't hurt though.